eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/ProxyConfiguration.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import com.azure.core.implementation.logging.ServiceLogger;
+
+import java.net.PasswordAuthentication;
+import java.net.Proxy;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class ProxyConfiguration implements AutoCloseable {
```

Comment:
Copy and paste the java docs from before. We have to ship this with documentation.

Reply:
Don't find a Java Doc from before. Will create one instead

Reply:
The javadoc doesn't exist for the class, but the other public methods exist. (ie. the constructor, SYSTEM_DEFAULTS).
https://github.com/Azure/azure-sdk-for-java/blob/6f072338d0fa4370171eafcd97dc95673fec4b63/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/ProxyConfiguration.java